src/types.ts

```diff
@@ -269,3 +269,8 @@ export interface APIAccessChangeEvent {
 export interface OrderedItem {
     order?: number;
 }
+
+export interface IFrameDimensions {
+    height: number;
+    width: number;
```

Comment:
Based on the discussion we had, I decided to go with both height and width since it seems like we might want to use this event for things other than modals. Like, I believe Rami mentioned something about custom widgets.

It's not a requirement that we keep both of these. We only need height for modals. So, if we'd prefer to not expand the API too much, we can drop the width.